clarifaipyspark/client.py

```diff
@@ -22,6 +24,12 @@ def __init__(self, user_id: str, app_id: str, pat: str = None):
     self.app_id = app_id
     self.app = App(user_id=user_id, app_id=app_id, pat=pat)
     super().__init__(user_id=user_id, app_id=app_id, pat=pat)
+    if pat:
+      self.metadata =  (("authorization", "Key %s" % pat), (REQUEST_ID_PREFIX_HEADER,
+                                                        REQUEST_ID_PREFIX))
```

Comment:
Why the PAT change? I see it was not there previously.
Also why do we add the prefix-header only when PAT is there? It should be added always?